.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
 
 script:
   - make test
-  - make xlang
+  - make xlang-fresh
```

Comment:
Looks like `docker-compose down` will be coming in `1.6`, which we could potentially run after failures...? Not quite sure how that would work.